sky/services/dynamic/BUILD.gn

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+source_set("embedder") {
+  sources = [
+    "dynamic_service.h",
+    "dynamic_service_embedder.cc",
+    "dynamic_service_embedder.h",
+    "dynamic_service_macros.h",
+  ]
+
+  deps = [
+    "//mojo/public/c/environment:environment",
+    "//mojo/public/c/system:system",
+  ]
+}
+
+source_set("dylib") {
+  sources = [
+    "dynamic_service.h",
+    "dynamic_service_dylib.cc",
+    "dynamic_service_dylib.h",
+    "dynamic_service_macros.h",
+  ]
+
+  deps = [
+    "//mojo/public/c/system:system",
+    "//mojo/public/c/system:system",
```

Comment:
this is doubly redundant.  you can just say "//mojo/public/system" as //foo/bar is defined to be the same thing as //foo/bar:bar and you have this line listed twice